bin/runtests

```diff
@@ -281,34 +284,23 @@ function get_phpunit_path()
 
 function init_bundle()
 {
-    if (!$console = get_test_console_path()) {
-        return;
-    }
-
-    $process = exec_cmd(sprintf(
-        PHP_BINARY . ' %s %s',
-        $console,
-        'debug:container doctrine.orm.entity_manager'
-    ), false, false);
-
+    $process = exec_sf_cmd('debug:container doctrine.orm.entity_manager', false, false);
     // bundle does not use Doctrine ORM
     if ($process->getExitCode() !== 0) {
         return;
     }
 
     write_info('Doctrine ORM detected, updating the schema for the bundle.');
 
-    exec_cmd(sprintf(
-        PHP_BINARY . ' %s %s',
-        $console,
-        ' doctrine:schema:update --force'
-    ));
+    $process = exec_sf_cmd('doctrine:schema:update --force');
+    if ($process->getExitCode() !== 0) {
+        exit($process->getExitCode());
+    }
 
-    exec_cmd(sprintf(
-        PHP_BINARY . ' %s %s',
-        $console,
-        ' sulu:document:initialize --purge --force --ansi'
```

Comment:
I think we don't need to catch this. Aslong as the tests run as expected it is not required that a bundle uses the document manager bundle.